Some UI enhancements and typo fixes

romanogit opened the pull request:
Suggested fixes
CardZoom.jsx: added a key to DIV loop to avoid browser warning in DEV mode
DeckSummary.jsx: add a divStyle at the bottom to allow better resizing of deck's card list
PlayerRow.jsx/gameboard.less: handling purged pile the same way as others to allow manual move of purge cards
Profile.jsx: fixed an issue which was not allowing email to be validated and causing page warning
MotherNorthelle.js: standardized aember wording to amber, since nowhere else aember is spelled this way
NotFinishedWithYou.js: fixed a typo

cryogen commented on client/Components/Decks/DeckSummary.jsx:
```diff
@@ -157,6 +163,7 @@ class DeckSummary extends React.Component {
                         { cardsToRender }
                     </div>
                 </div>
+                <div style={ divStyle } />
```
This should be in the .css/.less if it's not dynamic

romanogit replied:
Ok, I will fix it.

romanogit commented:
Changed the way I fixed #283 to also handle #342 :
![image](https://user-images.githubusercontent.com/2104075/63530605-2e976100-c4dd-11e9-8ec8-8adfe2a5ed5c.png)

romanogit commented:
Closing this pull request as it has too many commits and it is holding trivial changes
